scripts/generate_dump

```diff
@@ -294,6 +308,19 @@ main() {
         save_file $sai_dump_filename sai_sdk_dump true
     fi
 
+    local asic="$(/usr/local/bin/sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)"
+    if [ "$asic" = "broadcom" ]; then
+        save_cmd "bcmcmd -t5 version" "broadcom.version"
+        save_cmd "bcmcmd -t5 soc" "broadcom.soc"
+        save_cmd "bcmcmd -t5 ps" "broadcom.ps"
+    fi
+
+    if $GREP -qi aboot /host/machine.conf; then
```

Comment:
>$GREP -qi aboot /host/machine.conf [](start = 7, length = 34)

Check $aboot_platform?

Reply:
Are you suggesting I source /host/machine.conf and then check the $aboot_platform environment variable?

Reply:
Either way:
1. source /host/machine.conf, and check $aboot_platform.
2. grep "aboot_platform="

---
In reply to: [197968999](https://github.com/Azure/sonic-utilities/pull/272#discussion_r197968999) [](ancestors = 197968999)

Reply:
I implemented option 2. The Arista platform check is now more explicit.